kassy

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+# kassy - A bash CLI tool for running kassy in docker
+# Written By: Matthew Hartstonge <matt@mykro.co.nz>
+
+# Soak in user params
+ARGS="$@"
+
+function usage() {
+  echo "usage: kassy [--all] [--fb] [--skype] [--slack] [--test]               "
```

Comment:
This is true, I could scrape it back to simply passing in what normally would be passed via cli to 'node main.js' ? Essentially you only lose the benefit of breaking into the container via cmd command for debug purposes...